DESCRIPTION

```diff
@@ -53,7 +53,6 @@ Suggests:
     dplyr,
     ggplot2,
     gsDesign,
-    gsDesign2 (>= 1.1.3),
```

Comment:
This needs to be `gsDesign2 (>= 1.1.3.4),`. The CI jobs are failing because {gsDesign2} is not getting installed (because it isn't in Suggests). You can think of `Remotes` as purely supplemental. It is indicating which of your existing dependencies to obtain from GitHub.

Reply:
Hadley also has a good [explanation](https://r-pkgs.org/dependencies-in-practice.html#depending-on-the-development-version-of-a-package) of the `Remotes` field in his book.

Reply:
Thank you both! I added `gsDesign2(>= 1.1.3.4)` into the `Suggests` of the DESCRIPTION, and there are no `Remotes`, but the CI jobs failed. Could you please specify if there is a way I could get the CI jobs to run successfully?

Reply:
> there are no `Remotes`

Why did you remove the section `Remotes`? You need the development version of {gsDesign2} installed from GitHub, right?

To do this, you need _both_ the dependency requirement in `Suggests`:

```
Suggests:
    gsDesign2 (>= 1.1.3.4),
```

_and_ the section `Remotes` to download from the GitHub repository:

```
Remotes:
    Merck/gsDesign2
```

The error you are currently receiving is because you require {gsDesign2} 1.1.3.4, but that is not yet available from CRAN:

```
* deps::.: Can't install dependency gsDesign2 (>= 1.1.3.4)
```